apps/sms/views/conversation/js/conversation.js

```diff
@@ -407,6 +407,7 @@ var ConversationView = {
   },
 
   getIdIterator: function conv_getIdIterator() {
+    // TODO: Will need to replace with Conversation service method.
```

Comment:
Caching the ids might increase the complexity for maintenance since the messages could be updated frequently and we have to make sure this cache is synced with messages in service all the time. That means we need to update the cache while receiving the message event and I'm not sure if it's a proper way to do so in conversation view.